### PR TITLE
Resolved App crash Fixes #585

### LINKED ISSUE #585: App Crashing when clicked on History on HomeActivity
### Description
  App Crashes when clicked on History, Add Password, Remove Password, Add Images and some others on HomeActivity.
Error: IndexOutofBoundException 

Also appropriate menu is not getting highlighted in Navigation Drawer.
### Steps to reproduce
* Open App. 
* And click on any of the above option
### Actual behaviour 
App Crashes

### Expected behaviour
Should open respective fragment.

### PR DESCRIPTION
**Pull request by jddeep**
# Description
The App opens successfully on clicking on any card from HomeFragment.
## (Example) History Activity onClick 
screenshot_20190304-224312

Fixes #585 

## Type of change
Just put an x in the [] which are valid.
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
- [x] `./gradlew assembleDebug assembleRelease`
- [x] `./gradlew checkstyle`

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by jddeep** [comment]
@Swati4star please review.

**Comment by Swati4star** [comment]
@vyankatesh24 review this PR

File: app/src/main/java/swati4star/createpdf/fragment/HomeFragment.java [hunk 1]
```diff
@@ -27,6 +30,7 @@
 import static swati4star.createpdf.util.Constants.REMOVE_PAGES;
 import static swati4star.createpdf.util.Constants.REMOVE_PWd;
 import static swati4star.createpdf.util.Constants.REORDER_PAGES;
+import static swati4star.createpdf.util.Constants.appName;
```
Comment (vyankatesh24) [inline app/src/main/java/swati4star/createpdf/fragment/HomeFragment.java]: Remove these newly added unused import statements.

**Comment by jddeep** [comment]
@vyankatesh24 @Swati4star All changes have been seen into. Please review the PR. Thanks.

**Pull request merged**